source/concepts/applications.rst

```diff
@@ -0,0 +1,717 @@
+  could not be loaded is undefined.
+* ``playback/next() -> None``: if the server allows switching between molecular
+  systems, this command requests the simulation with the next index to be
+  loaded as the current simulation. The server is free to cycle through the
+  available systems or ignore the command when the current system is the last
+  available one. The behaviour when the system fails to load is undefined.
+
+.. note::
+
+   There is no command defined to toggle between playing and pausing the frame
+   generation. This is on purpose as such a toggle command would be prone to
+   race conditions when multiple clients call play/pause commands close to each
+   other in time.
+
+.. warning::
+
+   The playback commands do not define any error handling. The commands to
+   switch among molecular systems can be silently ignored and a failure to load
+   a system, which is a probable event, has no defined behaviour.
+
+Simulation box for multi user use cases
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If the trajectory application is used in combination with the :ref:`multiplayer
+application <multiplayer-application>`, it can share where the simulation box
+should be placed relative to the avatar.
+
+The clients or the server can set the ``scene`` key in the :ref:`shared state
+<state-service>`. The value under that key is a list of numbers that merges
+position of the box's origin, its rotation as a quaternion, and the scaling
+compared to the default box size in each dimension. These are expressed in the
+:ref:`server coordinate system <multiplayer-coordinate-systems>`.
+
+By default:
+
+* the origin of the simulation space is set at the origin of the server space
+  (`i.e.` the position is ``[0, 0, 0]``);
+* the Y and Z axes of the simulation space match the Y and Z axis of the server
+  space, respectivelly; the X axis of the simulation space is reversed compared
+  to the one of the server space, so positive X values in simulation space
+  correspond to negative X values in the server space. This corresponds to a
+  ``[0, 0, 0, 1]`` quaternion.
+* 1 nanometer in simulation space corresponds to 1 meter in server space
+  (`i.e.` the scale is ``[1, 1, 1]``).
+
+The default ``scene`` value is therefore ``[0, 0, 0, 0, 0, 0, 1, 1, 1, 1]``.
+
+.. note::
+
+   The server space is Y-up while the simulation space is Z-up. However, the
+   default orientation of the box matches the XY axes of both space so clients
+   are expected to represent the simulation Y-up. In cases where the up
+   orientation of the simulation space is meaningful, the simulation space must
+   be rotated by setting the ``scene`` key rather than by altering the default
+   orientation.
+
+.. warning::
+
+   The scale can be set to any value but it must be set to 3 identical positive
+   value for the simulation space to keep its aspect ratio and not be mirrored.
+
+
+.. _imd-application:
+
+The iMD application
```

Comment:
@hjstroud could you have a look at this section, please? I would like you to see if it makes sense, but also if it matches the source code: https://github.com/IRL2/nanover-protocol/blob/main/python-libraries/nanover-core/src/nanover/imd/particle_interaction.py

Reply:
I have added some initial comments and suggestions (see below), I hope these are useful! I will compare with the source code now and let you know if I find anything else.